airflow/plugins_manager.py

```diff
@@ -150,6 +152,8 @@ def load_entrypoint_plugins():
                 plugin_instance = plugin_class()
                 if callable(getattr(plugin_instance, 'on_load', None)):
                     plugin_instance.on_load()
+                    plugin_instance.plugin_source = "entrypoint_plugin"
+                    plugin_instance.plugin_path = str(entry_point.dist)
```

Comment:
Can we include `str(entry_point)` somehow? It was part of my original FR.

Reply:
Sure, Done.